check-file-age-pattern.py

```diff
@@ -17,14 +17,76 @@
     help="files pattern (give them between '' if you use *",
 )
 parser.add_argument(
-    "-w", "--warning", dest="warning", type=int, default=86400, help="TODO"
+    "-w",
+    "--warning",
+    dest="warning",
+    type=int,
+    default=86400,
+    help="minimum modification time in seconds to raise a warning",
 )
 parser.add_argument(
-    "-c", "--critical", dest="critical", type=int, default=86400 * 2, help="TODO"
+    "-c",
+    "--critical",
+    dest="critical",
+    type=int,
+    default=86400 * 2,
+    help="minimum modification time in seconds to raise a critical",
 )
+
+parser.add_argument(
+    "-a",
+    "--all",
+    dest="all",
+    action="store_true",
+    help="All files found must match minimal modification time",
+)
+
 current_time = time.time()
 
 
+def check_all(age_path_tuple_list):
+    # Need to check for a critical then we can test warnings.
+    for age, path in age_path_tuple_list:
+        if age > args.critical:
+            # f-string not possible because of compatibility versions
+            print(
+                path
+                + " modified "
+                + str(datetime.timedelta(seconds=int(age)))
+                + " ago."
+            )
+            sys.exit(2)
+    for age, path in age_path_tuple_list:
+        if age > args.warning:
+            # f-string not possible because of compatibility versions
+            print(
+                path
+                + " modified "
+                + str(datetime.timedelta(seconds=int(age)))
+                + " ago."
+            )
+            sys.exit(1)
+    sys.exit(0)
```

Comment:
Add a print to tell that all files are ok.

Reply:
Done